src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/DeleteCompleteFilesTask.java

```diff
@@ -80,58 +85,72 @@ private void processCompleteFiles(CloudBlobContainer container) {
 
         log.info(
             "Finished deleting complete files in container {}, deleted {} files, failed to delete {} files",
-            container.getName(),
+            container.getBlobContainerName(),
             successCount,
             failureCount
         );
     }
 
-    private boolean tryProcessCompleteEnvelope(CloudBlobContainer container, Envelope envelope) {
-        boolean deleted = false;
+    private boolean tryProcessCompleteEnvelope(BlobContainerClient container, Envelope envelope) {
 
-        String loggingContext = "File name: " + envelope.getZipFileName() + ", Container: " + container.getName();
+        String loggingContext = "File name: " + envelope.getZipFileName()
+            + ", Container: " + container.getBlobContainerName();
 
         try {
             log.info("Deleting file. {}", loggingContext);
 
-            CloudBlockBlob cloudBlockBlob = container.getBlockBlobReference(envelope.getZipFileName());
+            BlobClient blobClient = container.getBlobClient(envelope.getZipFileName());
 
-            if (cloudBlockBlob.exists()) {
-                Optional<String> leaseId = blobManager.acquireLease(
-                    cloudBlockBlob,
-                    container.getName(),
-                    envelope.getZipFileName()
+            if (Boolean.TRUE.equals(blobClient.exists())) {
```

Comment:
is _equals_ necessary?

Reply:
same question

Reply:
> is _equals_ necessary?

no

Reply:
removed

Reply:
Sonar did not liked without equals.

Reply:
https://sonarcloud.io/project/issues?id=Verification%3Abulk-scan-processor&open=AXPnlJAnYbrraxsQDZRm&pullRequest=1456&resolved=false&types=BUG